articles/Getting-started-with-windows-forms-using-c-#/index.md

```diff
@@ -0,0 +1,169 @@
+
+Now let's begin by adding the buttons and the textbox. First, we will add the numbers as buttons then we will add the C button the Calculating button the Operation buttons, and finally, the Textbox that will show the answer.
+
+**NOTE:** don't forget to change both the controls name and the text.
+
+![First_look1](/engineering-education/Getting-started-with-windows-forms-using-c-#/First_look1.png).
+
+### Inside the code
+Now let's start coding! First double click on any of the buttons to write your code inside it, let's start with the numbers.
+
+The following code inside each button will add the number to the `Output.Text`.
+
+```c#
+ private void Num0_Click(object sender, EventArgs e)
+        {
+            Output.Text += "0";
+        }
+
+        private void Num1_Click(object sender, EventArgs e)
+        {
+            Output.Text += "1";
+        }
+
+        private void Num2_Click(object sender, EventArgs e)
+        {
+            Output.Text += "2";
+        }
+
+        private void Num3_Click(object sender, EventArgs e)
+        {
+            Output.Text += "3";
+        }
+
+```
+
+Inside the `Dot_Click` we should discuss the case of multiple dots. In order To fix this problem, we should check the `Output.Text` if it has a dot already inside of it, if not we can normally add a dot to the text. **NOTE:** that we use the `try()` & `catch()` In order To prevent the user from doing invalid operations.
+
+```c#
+       private void Dot_Click(object sender, EventArgs e)
+        {
+            string w = Output.Text.ToString();
+            int len= w.Length;
+            if (Output.Text[--len] != '.')
+            {
+                Output.Text += ".";
+            }
+        }
+```
+
+To clear the `Output.Text` we simply do this.
+
+```c#
+private void Cbut_Click(object sender, EventArgs e)
+        {
+            Output.Text="";
+        }
+```
+
+Now let's move to the operations.
+
+```c#
+    private void Sum_Click(object sender, EventArgs e)
+        {
+            Output.Text += "+";
+        }
+
+        private void Minus_Click(object sender, EventArgs e)
+        {
+            Output.Text += "-";
+        }
+
+        private void Multi_Click(object sender, EventArgs e)
+        {
+            Output.Text += "*";
+        }
+
+        private void Div_Click(object sender, EventArgs e)
+        {
+            Output.Text += "/";
+        }
+```
+
+In this piece of code, we will deal with the `Sqrt()` & the `Pow()` methods.
+**NOTE:** that In order To use the Pow & Sqrt methods you need to import the Math class as follows `using static System.Math;`.
```

Comment:
If you using `NOTE:`, then you should not start the sentence after the `:` like it's a continuation of the Note.

You can rewrite as,

NOTE: In order to use ...

Reply:
Fix at all places in the article.